tests/integration/targets/change_request/tasks/main.yml

```diff
@@ -0,0 +1,246 @@
+- environment:
```

Comment:
The integration tests are missing a few checks. Each integration tests should contain the following checks:

 1. Creation in check mode.
 2. Normal creation.
 3. Normal creation with no parameter change from 2 to check change detection and state enforcement.
 4. Update in check mode.
 5. Normal update.
 6. Normal update with no parameter change from 5 to check change detection and state enforcement.
 7. Deletion in check mode.
 8. Normal deletion.
 9. Normal deletion again to make sure we do not fail if the record is already gone.
 10. Any other checks deemed relevant (can be interspersed between checks listed above).

We can use https://github.com/sensu/sensu-go-ansible/blob/master/tests/integration/molecule/module_etcd_replicator/converge.yml as a reference.